src/components/Chat/BottomSheetGif.js

```diff
@@ -48,15 +41,33 @@ const BottomSheetGif = React.forwardRef((props, ref) => {
     setIsLoading(false);
   };
 
+  const searchGIF = React.useCallback(
+    debounce((search) => {
+      onLoad(search);
+    }, 500),
+    []
+  );
+
+  const handleChangeText = (text) => {
+    setSearchText(text);
+    if (text?.trim()?.length > 2) {
+      searchGIF(text);
+    }
+  };
+
+  const handleOnClearText = () => {
+    setSearchText('');
+  };
+
   React.useEffect(() => {
-    onLoad();
+    onLoad('');
   }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The modification of the `useEffect` hook to call `onLoad` with an empty string as the initial search value is a good change. It ensures that the featured GIFs are loaded when the component mounts, even if there's no search query.
```diff
+ React.useEffect(() => {
+   onLoad('');
+ }, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->